pkg/controller/kogitoruntime/kogitoruntime_controller_test.go

```diff
@@ -78,8 +78,8 @@ func TestReconcileKogitoRuntime_Reconcile(t *testing.T) {
 	assert.True(t, framework.GetEnvVarFromContainer("NAMESPACE", &deployment.Spec.Template.Spec.Containers[0]) == instance.Namespace)
 	assert.Equal(t, "kogito-service-viewer", deployment.Spec.Template.Spec.ServiceAccountName)
 	assert.Len(t, deployment.Spec.Template.Spec.Volumes, 2) // #1 for property, #2 for downward api
-	// command to register protobuf
-	assert.Equal(t, deployment.Spec.Template.Spec.Containers[0].Lifecycle.PostStart.Exec.Command, podStartExecCommand)
+	// command to register protobuf does not exist anymore
+	assert.Nil(t, deployment.Spec.Template.Spec.Containers[0].Lifecycle)
```

Comment:
shouldn't we also remove it from images?

Reply:
Yes, I think @Kevin-Mok has a PR ready.

Reply:
Yes. I just wanted to try fixing my cluster first to test the s2i build, but I will submit it after that.